spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/src/main/java/org/springframework/cloud/data/module/deployer/lattice/ReceptorProcessStatusMapper.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.data.module.deployer.lattice;
+
+import org.cloudfoundry.receptor.commands.ActualLRPResponse;
+
+import org.springframework.cloud.data.module.ModuleStatus;
+import org.springframework.util.StringUtils;
+
+/**
+ * @author Michael Minella
+ */
+public class ReceptorProcessStatusMapper {
```

Comment:
Do we even need these methods to be extracted out? It seems that they are used exclusively by the 2 ModuleDeployer impls - why not keep the mapping logic encapsulated?